### PR TITLE
refactor(fe): refactor public directory

### PR DESCRIPTION
**Pull request by youznn**
### Description

- `public` : assets 모음
    - kebab case 네이밍 컨벤션 지키기
    - `logos` :  각 회사 및 서비스 로고
    - `icons` : 아이콘 라이브러리에 없는 아이콘
    - `banners` : 배너에서 사용되는 이미지

로 리팩토링을 진행하였습니다 

closes TAS-1027

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md)
- [x] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#pr-and-branch) and follow the [Commit Convention](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#commit-convention)
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

File: apps/frontend/app/(client)/(main)/page.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const slides = [
     topTitle: 'Welcome to',
     bottomTitle: 'CODEDANG',
     sub: 'Online Judge Platform for SKKU',
-    img: '/codedang-carousel.png',
+    img: '/logos/codedang-carousel.png',
```
Comment (eunnbi) [inline apps/frontend/app/(client)/(main)/page.tsx:14]:
```suggestion
    img: '/banners/codedang-carousel.png',
```

File: apps/frontend/app/(client)/(main)/page.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ const slides = [
     topTitle: 'SKKUDING',
     bottomTitle: 'Beta Service',
     sub: "Feel free to contact us if there's any bug",
-    img: '/bug-carousel.png',
+    img: '/banners/bug-carousel.png',
```
Comment (eunnbi) [inline apps/frontend/app/(client)/(main)/page.tsx:32]:
```suggestion
    img: '/banners/bug.png',
```

File: apps/frontend/components/Cover.tsx [hunk 3]
```diff
@@ -13,9 +13,9 @@ const bgColors: { [key: string]: string } = {
 }
 
 const icons: { [key: string]: string } = {
-  problem: '/codedang-icon.png',
-  notice: '/notice-icon.png',
-  contest: '/contest-icon.png'
+  problem: '/logos/codedang.png',
+  notice: '/logos/notice.png',
+  contest: '/logos/contest.png'
```
Comment (eunnbi) [inline apps/frontend/components/Cover.tsx:18]: 이 경로에 이미지가 없는 것 같아요!

Comment (eunnbi) [inline apps/frontend/components/Cover.tsx:18]:
```suggestion
  problem: '/banners/codedang.png',
  notice: '/banners/notice.png',
  contest: '/banners/contest.png'
```

Comment (youznn) [reply apps/frontend/components/Cover.tsx:18]: 꼼꼼한 코드리뷰 감사합니다..!! 꼼꼼히 한다고 했는데 놓친 부분이 많네요 😥

**Review by B0XERCAT — APPROVED** [review]

**Pull request merged**